packages/ui/bundlewatch.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "files": [
-    { "path": "./dist/components/sign-in.js", "maxSize": "40kB" },
-    { "path": "./dist/components/sign-up.js", "maxSize": "40kB" }
+    { "path": "./dist/components/sign-in.js", "maxSize": "80kB" },
+    { "path": "./dist/components/sign-up.js", "maxSize": "80kB" }
```

Comment:
Bumped for now. Localization file bumped over the previous 40kb.

Reply:
Ooof that's pretty big. Thinking aloud; what if we reduce the number of localisations by default and make them opt-in instead?